### DISCUSSION TITLE
How to breadcrumb to current nested folder?

### DISCUSSION THREAD

**Discussion by drmikecrowe** [SilentVoid13/Templater#585] [Help]
I'm experimenting with a folder structure like:

* Work
  * Client A
    * Journal
       * 2022
         * 2022-03
            * 2022-03-16
            * 2022-03-15

I would like a folder template that sets the `2022-03-16` breadcrumb for tomorrow to create that breadcrumb in the right place.  How would I go about that?

**Comment by JeromeZephyr** [comment]
You could do it with: 
- [tp.file - Templater](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#tpfilefolderrelative-boolean--false)
- `<% tp.date.now("YYYY-MM-DD",1,tp.file.title,"YYYY-MM-DD") %>`

Would do something like this:
```
[[<% tp.file.folder(false) %>/<% tp.date.now("YYYY-MM-DD",1,tp.file.title,"YYYY-MM-DD") %>|<% tp.date.now("YYYY-MM-DD",1,tp.file.title,"YYYY-MM-DD") %>]]
```
